### PR TITLE
Enable Drag Drop Upload

### PR DESCRIPTION
**Pull request by andreknieriem**
Enabled Drag&Drop Upload from Local to Server.

### CODE REVIEW

File: lib/views/tree-view.js [hunk 1]
```diff
@@ -234,6 +234,93 @@ class TreeView extends ScrollView {
     this.getTitle = () => 'Remote-FTP';
   }
 
+	TreeView.prototype.handleEvents = function() {
```
Comment (icetee) [inline lib/views/tree-view.js:247]: Please use ES6 code style / guide. Install [eslint](https://atom.io/packages/linter-eslint) to write a bug-free code to Atom.

Error: `Parsing error: Unexpected token . (fatal)`

**Comment by icetee** [comment]
Thanks for PR, but it does not run. Code is incorrect and you are using ES2015.
Please no compare master branch. There is [feature/drag_and_drop](https://github.com/icetee/remote-ftp/tree/feature/drag_and_drop) branch.

**Comment by andreknieriem** [comment]
Sorry, I must have forgotten something. I worked on it local in my atom packages folder. Will try to fix the bugs. In my version everything works fine and I don't need filezilla anymore.

**Comment by icetee** [comment]
I deleted it, please send it again.

**Pull request merged**